src/index.js

```diff
@@ -38,13 +39,16 @@ const run = async () => {
 			await git.clone()
 			await git.setIdentity(client)
 			await git.getBaseBranch()
-			await git.createPrBranch()
-
-			// Check for existing PR and add warning message that the PR maybe about to change
-			const existingPr = OVERWRITE_EXISTING_PR ? await git.findExistingPr() : undefined
-			if (existingPr && DRY_RUN === false) {
-				core.info(`Found existing PR ${ existingPr.number }`)
-				await git.setPrWarning()
+			
+			if (!SKIP_PR) {
+				await git.createPrBranch()
+
+				// Check for existing PR and add warning message that the PR maybe about to change
+				const existingPr = OVERWRITE_EXISTING_PR ? await git.findExistingPr() : undefined
```

Comment:
`existingPr ` needs to be declared outside of this if statement because it is used later.